Makefile

```diff
@@ -57,7 +57,6 @@ rdgo:
 # Composes an ostree
 .PHONY: rpmostree-compose
 rpmostree-compose: ${ROOT_DIR}/openshift.repo init-ostree-repo
-	cp -n ${ROOT_DIR}/RPM-GPG-* /etc/pki/rpm-gpg/
```

Comment:
Dropped because we are using hosts keys during composes correct?

Reply:
Yeah.

Reply:
Since https://github.com/openshift/os/pull/245